stock_valuation_account_manual_adjustment/models/stock_valuation_account_manual_adjustment.py

```diff
@@ -0,0 +1,177 @@
+                          required=True, states={'done': [('readonly', True)]})
+    document_date = fields.Date(
+        'Creation date', required=True, readonly=True,
+        default=lambda self: fields.Date.context_today(self))
+    user_id = fields.Many2one('res.users', 'Created by',
+                              readonly=True,
+                              states={'draft': [('readonly', False)]},
+                              default=lambda self: self.env.user)
+    post_date = fields.Datetime(
+        'Posting Date', states={'done': [('readonly', True)]},
+        help="Date of actual processing")
+
+    move_ids = fields.One2many(
+        comodel_name='account.move',
+        inverse_name='stock_valuation_account_manual_adjustment_id',
+        readonly=True)
+
+    company_id = fields.Many2one(
+        comodel_name='res.company', string='Company', readonly=True,
+        default=lambda self: self.env['res.company']._company_default_get(
+            'stock.inventory.revaluation'),
+        states={'draft': [('readonly', False)]})
+
+    @api.multi
+    def button_post(self):
+        self.post()
+        return True
+
+    @api.onchange("product_id")
+    def _onchange_product_id(self):
+        if self.product_id:
+            self.increase_account_id = \
+                self.product_id.categ_id and self.product_id.categ_id.\
+                property_inventory_revaluation_increase_account_categ
+            self.decrease_account_id = self.product_id.categ_id and \
+                self.product_id.categ_id.\
+                property_inventory_revaluation_decrease_account_categ
+
+    @api.model
+    def _prepare_move_data(self, date_move):
+
+        period = self.env['account.period'].find(date_move)[0]
+
+        return {
+            'narration': self.remarks,
+            'date': date_move,
+            'ref': self.name,
+            'journal_id': self.journal_id.id,
+            'period_id': period.id,
+            'stock_valuation_account_manual_adjustment_id': self.id
+        }
+
+    @api.model
+    def _prepare_debit_move_line_data(self, move, amount, account_id, prod,
+                                      date_move):
+        return {
+            'name': '(%s) %s' % (self.name, prod.name),
+            'date': date_move,
+            'product_id': prod.id,
+            'account_id': account_id,
+            'move_id': move.id,
+            'debit': amount
+        }
+
+    @api.model
+    def _prepare_credit_move_line_data(self, move, amount, account_id,
+                                       prod,
+                                       date_move):
+        return {
+            'name': '(%s) %s' % (self.name, prod.name),
+            'date': date_move,
+            'product_id': prod.id,
+            'account_id': account_id,
+            'move_id': move.id,
+            'credit': amount
+        }
+
+    @api.multi
+    def post(self):
+        timenow = time.strftime('%Y-%m-%d')
```

Comment:
Why not use fields.Datetime.now()? You use it on https://github.com/OCA/stock-logistics-warehouse/pull/228/files#diff-ecb3b276206e9b56ac80d5cccbfe6518R167.